### DISCUSSION TITLE
Jump between Tabs

### DISCUSSION THREAD

**Discussion by hditano** [helix-editor/helix#6236] [Q&A]
Hello there:

I made the conversion to Helix-Editor, but for some reason I cannot find the way to switch between tabs!!

Thx!

**Comment by tnahs** [comment]
There's Goto mode:

`gn` - Go to next buffer
`gp` - Go to previous buffer

https://docs.helix-editor.com/keymap.html#goto-mode

There's also Space mode:

`<space> b` - Open buffer picker

https://docs.helix-editor.com/keymap.html#space-mode

**Reply by hditano** [reply]
Thx for your quick reply.

I'm looking for something that will keep my buffer on the same position and only the cursor is the one moving from left to right...not the buffer itself.

**Reply by cd-a** [reply]
I think you're looking for
`<space>w l` jump to right buffer
`<space>w h` jump to left buffer

This is used when you have multiple splits

**Reply by hditano** [reply]
Not really... that one.

it was C-w w


Thx @cd-a !!

**Comment by BukiOffor** [comment]
Commenting to come back to this

**Comment by zelds** [comment]
`:vnew` (or `Ctrl+w v`) and `Ctrl+w w`
